packages/components/src/components/Layout.tsx

```diff
@@ -1,10 +1,11 @@
-import React, { ReactNode, useEffect } from 'react'
+import { ReactNode, useEffect } from 'react'
+import { cn } from '../lib/utils.ts'
 
 interface LayoutProps {
   children: ReactNode
   className?: string
   progress?: number
-  error?: Error
+  error?: unknown
```

Comment:
I understand why you did this. But it feels less type-safe?

Reply:
fixed